examples/torch/pearl_ml1_push.py

```diff
@@ -0,0 +1,163 @@
+"""PEARL ML1 example."""
+
+import akro
+from metaworld.benchmarks import ML1
+import numpy as np
+
+from garage.envs import normalize
+from garage.envs.base import GarageEnv
```

Comment:
please use canonical imports for all of these -- that is, `from garage.envs import GarageEnv` etc.